index.html

```diff
@@ -8537,22 +8556,24 @@ <h3 id="Nature"><span class="wip">Nature - Use with Caution</span></h3>
                </td>
                <td>Tree of Life
                </td>
-               <td>Heal all allies within 25ft as much as you decide, then suffer half the
+               <td>Heal all allies within 20ft as much as you decide, then suffer half the
                   damage healed.
                </td>
                <td>A, 25ft, All within range (AoE)
                </td>
-               <td>2
+               <td>3
                </td>
             </tr>
             <tr>
                <td>8
                </td>
                <td>Summon Spirit of Autumn
                </td>
-               <td>Summon an autumn spirit within 100ft, any creature that is within 15ft of
+               <td>Summon an autumn spirit within 100ft, any ally that is within 15ft of
                   the spirit can resist negative conditions for 1 point less and any creature
-                  within 5ft automatically resists negative conditions. This persists until no
+                  within 5ft automatically resists negative conditions. 
```

Comment:
Read similar comment on the other spirit summon, mainly the continuation effect.